backend/compact-connect/lambdas/common-python/cc_common/data_model/client.py

```diff
@@ -196,6 +196,7 @@ def create_privilege(self, *, compact: str, jurisdiction: str, provider_id: str)
                                     'status': provider_data['status'],
                                     'dateOfExpiration': provider_data['dateOfExpiration'],
```

Comment:
Accessibility:
* This textinput element does not have a name available to an accessibility API. Valid names are: label element, title undefined, aria-label undefined, aria-labelledby undefined.
* This form field should be labelled in some way. Use the label element (either with a "for" attribute or wrapped around the form field), or "title", "aria-label" or "aria-labelledby" attributes as appropriate.

<img width="771" alt="image" src="https://github.com/user-attachments/assets/dbb87422-7169-426e-9ae8-016986d3aedb">

@jsandoval81